index.html

```diff
@@ -8,9 +8,87 @@
       href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;500;700&display=swap"
       rel="stylesheet"
     />
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.3/css/all.css" integrity="sha384-SZXxX4whJ79/gErwcOYf+zWLeJdY/qpuqC4cAa9rOGUstPomtqpuNWT9wdPEn2fk" crossorigin="anonymous">
+    <link rel="stylesheet" href="css/style.css">
   <title>Responsive</title>
 </head>
 <body>
-  <!-- Your code here! Good luck! -->
+  <div id="wrapper">
+    <header>
+      <div class="container">
+        <div class="header-wrap">
+          <div class="logo">
+            <a href="#">
+              <img src="img/logo.png" alt="logo">
+            </a>
+          </div>
+          <nav class="nav">
+            <ul>
+              <li>
+                <a href="#">HOME</a>
+              </li>
+              <li>
+                <a href="#section about">ABOUT</a>
+              </li>
+              <li>
+                <a href="#">SERVICES</a>
+              </li>
+              <li>
+                <a href="#">PRICING</a>
+              </li>
+              <li>
+                <a href="#">CONTACT</a>
+              </li>
+            </ul>
+          </nav>
+        </div>
+      </div>
+    </header>
+    <main>
+      <div class="container">
+          <div class="section-hero-wrapper" id="hero-section">
+              <section class="hero">
+                <h1>Proffesional Developers Health Care</h1>
+                <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Voluptates qui, neque minima quibusdam, necessitatibus id fuga</p>
+                <div class="button-holder">
+                  <button class="button">
+                    <i class="fas fa-heartbeat"></i>
+                  </button>
+                  <button class="button">
+                    <i class="fas fa-first-aid"></i>
+                  </button>
+                  <button class="button">
+                    <i class="fas fa-stethoscope"></i>
+                  </button>
+                </div>
+              </section>
+          </div>
+          <div class="about-section-wrapper">
+            <div class="image-holder">
+              <img src="img/tools.png" alt="tools">
+            </div>
+            <div class="about-holder" id="section about">
```

Comment:
Do not use spaces in names of ID's or classes.

Reply:
fixed